### ISSUE TITLE
Create method for conda dev installation (from master current)

### ISSUE THREAD

**Issue by kassin** [spacetelescope/cubeviz#104]

**Comment by drdavella** [comment]
I'm looking into this. It may take a day or two to get the conda recipe right before we can start offering this option to testers. They will need to run the following commands to create a test environment using `conda`:
```
$ conda create -n cubeviz-testing python=3.6
$ source activate cubeviz-testing
$ conda install --override-channels -c http://ssb.stsci.edu/conda-dev -c defaults cubeviz
```

Note that the environment name is arbitrary: it does not need to be `cubeviz-testing`.

After the environment is created and activated, testers should be able to update daily by running the following command:
```
conda update -n cubeviz-testing --all --override-channels -c http://ssb.stsci.edu/conda-dev -c defaults cubeviz
```

**Comment by brechmos-stsci** [comment]
**Pre-Configuration - if conda is not installed yet:**

Step 1.  Install conda (https://conda.io/docs/user-guide/install/macos.html)

Step 2. Create a conda environment:
```
$ conda create -n cubeviz-testing python=3.6 pyqt
```

Step 3: Go into the environment:
```
$ source activate cubeviz-testing
```

Step 4: Create the `requirements.txt` file:

```
numpy
git+https://github.com/glue-viz/glue
git+https://github.com/spacetelescope/specviz
git+https://github.com/spacetelescope/cubeviz
```

Step 5:  Use pip to install the requirements:
```
$ pip install -r requirements.txt
```

**To Run Cubeviz (After Pre-Configuration)**

Step 1: Go into the conda environment

```
$ source activate cubeviz-testing
```

Step 2: Run cubeviz
```
$ cubeviz
```

**Comment by drdavella** [comment]
While it is still useful to have up-to-date requirements on `conda-dev`, this issue was superseded by the work that was done in #107, which provides a simple way for alpha testers to create reproducible test environments.
